config/environments/production.rb

```diff
@@ -78,6 +78,9 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
+
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
```

Comment:
at the end of this file there is a huge comment that is no longer auto-generated by rails:
```ruby
  # Inserts middleware to perform automatic connection switching.
  # The `database_selector` hash is used to pass options to the DatabaseSelector
  # middleware. The `delay` is used to determine how long to wait after a write
  # to send a subsequent read to the primary.
  #
  # The `database_resolver` class is used by the middleware to determine which
  # database is appropriate to use based on the time delay.
  #
  # The `database_resolver_context` class is used by the middleware to set
  # timestamps for the last write to the primary. The resolver uses the context
  # class timestamps to determine how long to wait before reading from the
  # replica.
  #
  # By default Rails will store a last write timestamp in the session. The
  # DatabaseSelector middleware is designed as such you can define your own
  # strategy for connection switching and pass that into the middleware through
  # these configuration options.
  # config.active_record.database_selector = { delay: 2.seconds }
  # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
  # config.active_record.database_resolver_context = ActiveRecord::Middleware
  #                                                  ::DatabaseSelector::Resolver::Session
```

Reply:
should I remove these comments then?

Reply:
I think so, because we didn't add them ourselves it's an autogenerated thing that is no longer autogenerated (for some reason)